File: ESCOM/pc/SistemasDistribuidos/proyecto3/EJERCICIO/ejercicio200217/numerico.c
#include <stdio.h>
#include <math.h>


#define Cal 1e6 *3


int main ()
{
	int i=0;

	double cose=0, seno=0 ,tang=0,loga=0,raiz=0;
	 while (i<Cal)
	 {
	 	cose+=cos(i);
	 	seno+=sin(i);
	 	tang+=tan(i);
	 	loga+=log10(i);
	 	raiz+=sqrt(i);
	 		i++;
	 }

return 0;
}

File: ESCOM/pc/SistemasDistribuidos/proyecto3/EJERCICIO/ejercicio200217/numerico.java


public class numerico
{
	static final double MAX=3000000;

	public numerico()
	{
		double cose=0;
		double seno=0;
		double tang=0;
		double loga=0;
		double raiz=0;
		double i=0;



		while(i<MAX)
		{
			cose+=Math.cos(i);
			seno+=Math.sin(i);
			tang+=Math.tan(i);
			loga+=Math.log10(i);
			raiz+=Math.sqrt(i);
			i++;

		}
	}
		public static void main (String[] args) 
		{
			numerico Numerico= new numerico();
		
		} 
}